node/middlewares/sitemap.ts

```diff
@@ -9,6 +9,13 @@ const updateRouteList = async (ctx: Context, route: Route[]) => {
   }
 }
 
+const xmlSitemapItem = (loc: string) => {
+  return `<sitemap>
```

Comment:
try indenting the code in the next line so it's easier to read, like so:
```suggestion
  return `
  <sitemap>
```